astropy/modeling/math_functions.py

```diff
@@ -14,16 +14,25 @@
                "arctanh", "deg2rad", "rad2deg"]
 
 
-math_ops = ["add", "subtract", "multiply", "divide", "logaddexp", "logaddexp2",
+math_ops = ["add", "subtract", "multiply", "logaddexp", "logaddexp2",
             "true_divide", "floor_divide", "negative", "positive", "power",
-            "remainder", "mod", "fmod", "divmod", "absolute", "fabs", "rint",
+            "remainder", "fmod", "divmod", "absolute", "fabs", "rint",
             "exp", "exp2", "log", "log2", "log10", "expm1", "log1p", "sqrt",
-            "square", "cbrt", "reciprocal"]
+            "square", "cbrt", "reciprocal", "divide", "mod"]
 
 
 supported_ufuncs = trig_ufuncs + math_ops
 
 
+# These names are just aliases for other ufunc objects
+# in the numpy API.  The alias name must occur later
+# in the lists above.
+alias_ufuncs = {
+    "divide": "true_divide",
+    "mod": "remainder",
```

Comment:
Modulo or remainder, hmmm. Is there a clear preference out there. Perhaps we should do a mini language survey. Well, to some these are not the same! https://rob.conery.io/2018/08/21/mod-and-remainder-are-not-the-same/